docs/content/documentation/systems/Kernels/phase_field/KKSMultiACBulkF.md

```diff
@@ -1,8 +1,58 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # KKSMultiACBulkF
 !description /Kernels/KKSMultiACBulkF
 
+KKS Allen-Cahn kernel for the terms without a direct composition dependence.
+
+### Residual
+
+$$
+R=-\frac{dh}{d\eta}(F_a-F_b)+w\frac{dg}{d\eta}.
```

Comment:
For the 3-phase KKS model, if the non-linear variable is $\eta_1$
$$
R = \left(\frac{dh_1}{d\eta_1} F_1 + \frac{dh_2}{d\eta_1} F_2 + \frac{dh_3}{d\eta_1} F_3 + W_i \frac{d g}{d \eta_1}  \right)
$$
where $h_i$ is the interpolation function for phase $i$ defined in Folch and Plapp, Phys. Rev. E, 72, 011602 (2005), $W_i$ is the energy barrier height, and  $g$ is the double-well function.